content/verification/address-validation/release-notes/2024-05-15.md

```diff
@@ -0,0 +1,11 @@
+We released version 3 (v3) of address validation. Updates include:
+- `/validate` and `/candidate` endpoints
```

Comment:
```suggestion
- Adding `/validate` and `/candidate` endpoints.
```

Reply:
these endpoints existed in v1 of their API, so I don't think we can say they are being added here.  v3 basically takes functionality of v1 and v2 and combines them into v3 and describes the major changes such as congressional district and countrycode ISO style that is now different in v3 vs v1

Reply:
Got it, I can remove this comment.